docs/tempo/website/troubleshooting/_index.md

```diff
@@ -7,7 +7,6 @@ weight: 470
 
```

Comment:
Thoughts on changing the title? Maybe: `Troubleshooting missing traces` -> `Troubleshooting`

Reply:
agree :+1: